### ISSUE TITLE
Old profit calculations are changing frequently and becoming wildly inaccurate

### ISSUE THREAD

**Issue by elitriona** [bTayFla/PtTracker#137]
<img width="812" alt="screen shot 2018-05-03 at 8 40 33 am" src="https://user-images.githubusercontent.com/34076601/39587415-4f3b53ea-4eae-11e8-8349-e8729fef3100.png">

That February number is completely wrong. I never had any negative numbers that entire month. Furthermore, the number is changing a bunch through the day. For example, yesterday it was +1.2 % then later in the day it was -.66 %. It's all over the place. Those are all old sales from months ago, but the calculation is changing all the time in wild swings!

**Comment by LiamAird** [comment]
Did you switch Base Currency you trade with From Feb to May?

**Comment by elitriona** [comment]
No, however, I am running two bots on the same VPS (one ETH and one BTC). Each bot operates on different Binance accounts.

**Comment by LiamAird** [comment]
https://discord.gg/8tbjgW

Please join Tracker Support and we can try to sort it out.

**Comment by cryptokerr** [comment]
having the same issues on all 3 bots

random changes to the monthly profits % and also daily average showing minus and plus at times

**Comment by elitriona** [comment]
I tried getting support on the discord. Someone asked for my database file to review, and they looked at it, but I never really got any answer of what was going on. The problem still persists, and it's made using tracker pretty much worthless for me at the moment.

**Comment by cryptokerr** [comment]
i was told my json file was corrupt on transfer but this started happening
just lately for me after running fine on 3 bots. Now all 3 of my bots are
showing crazy figures

**Comment by LiamAird** [comment]
This seems to happen if you used PTT and PT before the PT2.0, where the data format was different.

**Comment by elitriona** [comment]
Indeed. Might there be something in the works to resolve the issue for those who are affected?

**Comment by cryptokerr** [comment]
we are pretty aware of the problem, just need a fix :-)

Kind regards,

Willy Kerr
KTA Founding Director &
Event Manager



Tel: +66895292050
Skype: kiteboardtourasia
Visit KTA on Facebook <http://www.facebook.com/kiteboardtourasia> - Visit
Our Website <http://www.kiteboardtour.asia>

**Comment by symvar** [comment]
I have read the comments above and i'm happy that you guys are aware of the issue. I just wanted to say that i have the same thing (and possibly more) going on. I have seen February gain % swings from -1200% to +3000% and some stuff in between. Also, the averages for other months (March, April, May) seem incorrect when comparing to my own tracking sheet. 
It also looks like PPT thinks my TCV didnt change for he past two weeks, its exactly the same each day while i know the values are different each day. 

Cheers ;)
